player/lua/select.lua

```diff
@@ -48,6 +48,8 @@ mp.add_key_binding(nil, "select-playlist", function ()
         if entry.playing then
             default_item = i
         end
+
+        playlist[i] = (entry.playing and "●" or "○") .. " " .. playlist[i]
```

Comment:
I would really prefer to have color/shadow/border highlight for current item, instead of bloating everything with ● and ○. Note that we are not in console anymore, we can use ASS.

Reply:
I went with `styles.warn` because it's similar to the yellow of the selected item, better suggestions are welcome. The terminal style is just bold.

Reply:
Changed to the same color as the selected item but not bold.

Reply:
Little bit similar looking. I was thinking something more subtle like "hidden items" color. But it is fine, I'm not so much into colors, I can get used to it.

Reply:
/cc @na-na-hi

Reply:
For example using a black shadow helps out .
To get a black border behind search results with a little white shadow for better search results visibility in script-modules/scroll-list.lua near the top
`list_style = [[{\q2\fs20\c&Hffffff&}]],`
change to
`list_style = [[{\backcolour&H000000&\shad1\bord4\q2\fs20\c&Hffffff&}]],`


<img width="1089" alt="image" src="https://github.com/user-attachments/assets/37d2ecc6-06c3-4c78-a0da-f1649dca07c3">

compared to select.lua
<img width="1112" alt="image" src="https://github.com/user-attachments/assets/c62d4942-b488-4a34-9b53-7c34d554ffab">

showing black border with white text in youtube-live chat
<img width="922" alt="image" src="https://github.com/user-attachments/assets/990830ca-dd9a-4d94-9612-1c78c80b39e3">


This one I use for renaming files but it would be nice if there could be a small help section on top with a couple shortcut keys used for console. Most important one I think is Ctrl+u
<img width="476" alt="image" src="https://github.com/user-attachments/assets/c3ff652e-f603-4784-b04d-0d00e0c3f7f1">

Reply:
You can increase `console-border_size`. I agree that the default is completely unreadable which is why I tried to increase it in https://github.com/mpv-player/mpv/pull/14088

Reply:
And in the end, we didn't scale with video, nor we made the size bigger.

Reply:
They have circles in case multiple tracks are selected.

Reply:
I added circles only to tracks precisely because multiple tracks can be selected but not multiple playlist entries or anything else, it is not to arbitrarily make separate views different or whatever.